packages/zowe-explorer/README.md

```diff
@@ -434,3 +435,17 @@ As a result, VS Code extensions like Zowe Explorer, Zowe Explorer CICS Extension
 ![Allocate Like](/docs/images/ZE-allocate.gif?raw=true "Allocate Like")
 
 [Back to sample list](#sample-use-cases)
+
+---
+
+### Search a Partitioned Data Set
+
+1. Navigate to the **Side Bar**
+1. Open the **DATA SETS** bar.
+1. Right-click a partitioned data set and select the **Search PDS Members** option.
+1. When prompted, type in the string to search the PDS members for.
+   - A confirmation dialog will appear if there are a large number of members to search.
```

Comment:
Try to avoid future tense: "... dialog appears if there is a large number of ... "